messages/en.json

```diff
@@ -16,5 +16,11 @@
   "editor.product-details.highlights.chooseDefaultSpecification": "Fields Specifications",
   "editor.product-details.thumbnailsPosition.title": "Thumbnails position",
   "editor.product-details.thumbnailsPosition.left": "Left",
-  "editor.product-details.thumbnailsPosition.right": "Right"
+  "editor.product-details.thumbnailsPosition.right": "Right",
+  "product-details.quantity": "Quantity",
+  "product-details.quantity-available": "{availableQuantity} available",
+  "editor.product-quantity-selector.title": "Product Quantity Selector component",
+  "editor.product-quantity-selector.description": "A component that allows user to select the amount of items to add to cart",
+  "editor.product-quantity-selector.maximumAvailableQuantity.title": "Maxiumum Available Quantity",
```

Comment:
It's not actually the maximum available quantity I would say it's the `warningQuantityThreshold`?

Reply:
looks like a better name indeed